Some drivers rely on some other part of the kernel to include
in.h but on older kernels this proves to not work. Explicitly
include that file. This fixes compilation of ath6kl on 2.6.27.
This patch will be sent upstream though.

--- a/drivers/net/wireless/ath/ath6kl/wmi.c
+++ b/drivers/net/wireless/ath/ath6kl/wmi.c
@@ -16,6 +16,7 @@
  */
 
 #include <linux/ip.h>
+#include <linux/in.h>
 #include "core.h"
 #include "debug.h"
 #include "testmode.h"
